app/models/podcast.rb

```diff
@@ -14,8 +14,18 @@ class Podcast < BaseModel
 
   acts_as_paranoid
 
+  before_validation :set_defaults
+
   scope :published, -> { where('published_at IS NOT NULL AND published_at <= now()') }
 
+  def set_defaults
+    self.enclosure_template ||= enclosure_template_default
+  end
+
+  def enclosure_template_default
+    "https://#{ENV['DOVETAIL_HOST']}/{slug}/{guid}/{original_filename}"
```

Comment:
Yeah, as I re-read this, the `enclosure_prefix` stuff you added makes sense.  But how `enclosure_template` fits in there is still a bit murky.  I may just need to revisit in the morning.